packages/welldone-wallet/src/lib/injected-welldone.ts

```diff
@@ -0,0 +1,37 @@
+import { Action } from "@near-wallet-selector/core";
+
+export interface WelldoneWalletParams {
+  iconUrl?: string;
+  deprecated?: boolean;
+}
+
+export interface RequestParams {
```

Comment:
`RequestParams`  and `WalletProvider` are extending interfaces, so it would be nice to remove unnecessary export and put them on top of each other before `WelldoneWalletState`

```
interface RequestParams {
    ...
}

interface WalletProvider {
    ...
}

export interface WelldoneWalletState {
    ...
}

```

Reply:
Thanks! I remove unnecessary export at `RequestParams`, but I left `WalletProvider` as it was since I import `WalletProvider` from `welldone.ts`